compose/snippets/src/main/java/com/example/compose/snippets/glance/GlanceSnippets.kt

```diff
@@ -868,6 +870,34 @@ object GlanceTheming {
     }
 }
 
+object GlanceInnerPadding {
+
+    /**
+     * Applies corner radius for views that are visually positioned [widgetPadding]dp inside of the
+     * widget background.
+     */
+    @Composable
+    fun GlanceModifier.appWidgetInnerCornerRadius(): GlanceModifier {
+        // [START android_compose_glance_innercornerradius]
```

Comment:
You may want to start this snippet with the @Composable part since you are returning something at the end of the function as part of the snippet, so it makes sense to see the full function definition.

Reply:
Updated!